src/test/java/org/openlmis/stockmanagement/controller/StockCardControllerTest.java

```diff
@@ -131,17 +130,32 @@ public void setupEvent() {
     event.setQuantity(10L);
   }
 
-  public void setupLot() {
-    lot = new Lot();
-    lot.setId(lotId);
+  public Lot setupLot(Long id)
+  {
+    Lot lot = new Lot();
+    lot.setId(id);
     lot.setProduct(defaultProduct);
-    lot.setLotCode("A1");
-    lot.setManufacturerName("Manu");
+    lot.setLotCode("code_" + id);
+    lot.setManufacturerName("Manufacturer_of_" + id);
     lot.setManufactureDate(new Date());
     lot.setExpirationDate(new Date());
-    event.setLot(lot);
+    //event.setLot(lot);
+    return lot;
+  }
+
+  //Associate two lots with the specified StockCard - one lot for which we have stockOnHand, and one for which we don't
+  public void addLotsToStockCard(StockCard card)
```

Comment:
Method name is not clear that it simply sets the stock card to have two lots; expecting that lots are added each time the method is called.